src/gui/transferlistfilterswidget.cpp

```diff
@@ -393,7 +395,15 @@ void TrackerFiltersList::downloadFavicon(const QString& url)
 
 void TrackerFiltersList::handleFavicoDownload(const QString& url, const QString& filePath)
 {
-    QString host = getHost(url);
+    QString host;
+
+    if (url.startsWith(GOOGLE_FAVICON_URL)) {
+        host = url.mid(url.indexOf(GOOGLE_FAVICON_URL) + GOOGLE_FAVICON_URL.size());
```

Comment:
Isn't url.indexOf(GOOGLE_FAVICON_URL) equal 0 and GOOGLE_FAVICON_URL.size() is enough.